dashboards/workspace/workspace_account_report.sp

```diff
@@ -43,78 +31,127 @@ dashboard "workspace_account_report" {
         display = "none"
       }
 
```

Comment:
The `workspace` column is set to `display = "none"`, but we don't have a `workspace` column; instead, we have `Workspace` column. Do we still need it?"

Reply:
not required, removed it. Thanks for catching this!